Makefile

```diff
@@ -108,6 +101,13 @@ coverage: test
 	cat $(COVERAGE_FILE) | grep -v "_mock.go" > $(COVERAGE_FILE).no-mocks
 	go tool cover -func=$(COVERAGE_FILE).no-mocks
 
+ginkgo:
+	mkdir -p $(CURDIR)/bin
+	GOBIN=$(CURDIR)/bin go install github.com/onsi/ginkgo/v2/ginkgo@latest
+
+test-e2e: ginkgo
```

Comment:
Let's rather have a `tests/e2e/Makefile` for these targets. I'm happy to also have an "alias" that invokes that target at this level, but the core logic should be in the tests folder.

Reply:
k will do

Reply:
done